DataProcessing/process.py

```diff
@@ -0,0 +1,96 @@
+# 1. Download the data
+# 2. process (if required)
+# 3. Output data by date tp /temp-output-directory/alternative/quiver/twitter/universe/{date}.csv
+# 4. Output data for each ticker to /temp-output-directory/alternative/quiver/twitter/{symbol}.csv
+
+import os
+from clr_loader import get_coreclr
+from pythonnet import set_runtime
+set_runtime(get_coreclr(os.path.join(os.path.dirname(os.path.realpath(__file__)), "config.json")))
+
+from AlgorithmImports import *
+from QuantConnect.Lean.Engine.DataFeeds import *
+AddReference("Fasterflect")
```

Comment:
Think we should push this section to it's own file in the template/sdk repo and just reuse it when required. I think ideally we should remove the dependency of the `config.json`. We can probably set the runtime providing it as a string versus the lib reading it from the file